server/routes/api.js

```diff
@@ -1,6 +1,6 @@
 var express = require('express');
 var router = express.Router();
-const articleDB = require('../dbClients/articlesDB')
+const categoryClient = require("../dbClients/categoriesDB");
 
```

Comment:
this is wrong - articlesDB is still needed by /articles. You need both articlesDB and categoriesDB (and all it categoriesDB like the file not categoriesClient)